Tutorial split (#145)
pretix/pretix-docs: docs/guides/email.md

# Email settings

pretix offers you many options for contacting your customers and attendees via email—both on the organizer level and on the event level.
You can use either the system-provided email server or your organization's own.
pretix comes with many pre-configured email messages that you can customize to your needs.
This article tells you how to set up your desired email server solution within pretix and how to customize email content.

## Prerequisites

Your account needs to be activated before you can use pretix to send out emails.
See the tutorial's section on [account activation](../tutorial/single-events/organizer-account.md#activation) for further information.

## How to

### General email settings

By default, the general email settings are handled on the organizer level.
Navigate to :navpath:Your organizer → :fa3-wrench: Settings → E-Mail:.
Click the :btn-icon:fa3-edit: Edit: button next to "Sending method" and choose one of the three options:

 1. **Use system default:** system-provided email server.

>  Emails will be sent from `support@pretix.eu`.

Reviewer: ```suggestion
 Emails will be sent from `noreply@pretixmail.com`.
```

Reviewer 2: > Emails will be sent from `support@pretix.eu`.

Ist das schon wieder ein GitHub-Anzeige-Problem, dass das für mich immer noch als die aktuelle Version angezeigt wird?

Author: Die Änderung gibt es schon lange auf `main`. Hat also keinen Sinn, die hier nochmal anzuwenden.

 Emails for customer account registration, email change, and password reset will have your organizer account's name in the name field.

>  All other emails with have the event name in the name field.

Author: Test
```suggestion
 All other emails have the event name in the name field.
```

 2. **Use system email server with a custom sender address:** system-provided email server.
 Emails will be sent from a custom email address.
 This requires adding the pretix application server to your mail server's SPF record (see [Using system email server with a custom sender address](email.md#using-system-email-server-with-a-custom-sender-address) below).
 3. **Use a custom SMTP server:** Use your organization's own SMTP server and fully customize mailing.

If you want to change the general email settings for an individual event, you can unlock them for that event.
Unlocking the settings is irreversible.
Once they are unlocked, changing email settings for your organizer does not change the settings for this particular event anymore.
Navigate to :navpath:Your event → :fa3-wrench: Settings → E-Mail: and click the :btn-icon:fa3-unlock: Unlock: buttons as required.

### Using system email server with a custom sender address

If you use an email address with your own domain as the sender address and do not use a custom SMTP server, you have to add the pretix application server to your SPF record.

> This is necessary to prevent your emails being misidentified as spam.

Reviewer: prevent someone from doing sth
```suggestion
This is necessary to prevent your emails from being misidentified as spam.
```
